rqt_py_common/package.xml

```diff
@@ -33,8 +33,6 @@
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend>
   <test_depend>ament_cmake_pytest</test_depend>
-  <test_depend>ament_flake8</test_depend>
-  <test_depend>ament_pep257</test_depend>
   <test_depend>python3-pytest</test_depend>
```

Comment:
This test dependency doesn't seem to be needed anymore?

Reply:
Which dependency are you talking about? python3-pytest?

Reply:
Yes.

Reply:
Removed